test/docfx.Test/lib/GitHubAccessorTest.cs

```diff
@@ -33,23 +33,21 @@ public async Task GetUserByLogin(string login, int? expectedId, string expectedN
-        [InlineData("OPS-E2E-PPE", "E2E_Contribution_DocfxV3", "e0f6bbdf1c8809562ca7ea1b3749660078143607", false, "OPSTestPPE", 26447601, "OPSTestPPE", new[] { "opse2etestingppe@outlook.com" })]
-        [InlineData("OPS-E2E-PPE", "E2E_Contribution_DocfxV3", "c2f754e529491f59a7ceaa1376308820ba05f586", false, "luyajun0205", 15990849, "luyajun0205", new[] { "v-yajlu@microsoft.com" })]
-        [InlineData("docascode", "docfx-test-dependencies", "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef", false, null, null, null, null)]
-        [InlineData("docascode", "contribution-test", "b2b280fbc64790011c7a4d01bca5b84b6d98e386", false, null, null, null, new[] { "51308672+disabled-account-osmond@users.noreply.github.com" })]
-        [InlineData("docascode", "this-repo-does-not-exists", "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef", true, null, null, null, null)]
-        public async Task GetUserByCommit(string repoOwner, string repoName, string commit, bool hasError, string login, int? id, string name, string[] emails)
+        [InlineData("docascode", "docfx-test-dependencies", "c467c848311ccd2550fdb25a77ef26f9d8a33d00", "OsmondJiang", 19990166, "Osmond Jiang", new[] { "xinjiang@microsoft.com" })]
+        [InlineData("OPS-E2E-PPE", "E2E_Contribution_DocfxV3", "e0f6bbdf1c8809562ca7ea1b3749660078143607", "OPSTestPPE", 26447601, "OPSTestPPE", new[] { "opse2etestingppe@outlook.com" })]
+        [InlineData("OPS-E2E-PPE", "E2E_Contribution_DocfxV3", "c2f754e529491f59a7ceaa1376308820ba05f586", "luyajun0205", 15990849, "luyajun0205", new[] { "v-yajlu@microsoft.com" })]
+        [InlineData("docascode", "docfx-test-dependencies", "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef", null, null, null, null)]
+        [InlineData("docascode", "contribution-test", "0000000000000000000000000000000000000000", null, null, null, null)]
+        [InlineData("docascode", "contribution-test", "b2b280fbc64790011c7a4d01bca5b84b6d98e386", null, null, null, new[] { "51308672+disabled-account-osmond@users.noreply.github.com" })]
+        [InlineData("docascode", "this-repo-does-not-exists", "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef", null, null, null, null)]
+        public static async Task GetUserByCommit(string repoOwner, string repoName, string commit, string login, int? id, string name, string[] emails)
         {
-            var (error, users) = await _github.GetUsersByCommit(repoOwner, repoName, commit);
+            var (error, users) = await s_github.GetUsersByCommit(repoOwner, repoName, commit);
 
             // skip check if the machine exceeds the GitHub API rate limit
-            if (!string.IsNullOrEmpty(_token))
+            if (!string.IsNullOrEmpty(s_token))
             {
-                if (hasError)
-                    Assert.NotNull(error?.Message);
-                else
-                    Assert.Null(error?.Message);
+                Assert.Null(error);
```

Comment:
why `repo not found` doesn't return error here? is that a break change?

Reply:
reverted